crashsimilarity/cli/signature_coherence.py

```diff
@@ -1,9 +1,14 @@
-# CLI INTERFACE THAT EVALUATES THE SIMILARITY BETWEEN STACK TRACES IN A GIVEN SIGNATURE.
+import pprint
+
 from crashsimilarity.downloader import SocorroDownloader
-from crashsimilarity import crash_similarity
 import sys
 import argparse
 
+from crashsimilarity.models.gensim_model_wrapper import Doc2vecModelWrapper
+from crashsimilarity.models.similarity.doc2vec_similarity import Doc2VecSimilarity
+from crashsimilarity.models.wmd_calculator import WMDCalculator
+from crashsimilarity.utils import StackTracesGetter
```

Comment:
I would prefer the users of the module to not worry about importing all the internal modules, but just an external interface. We can fix this in the future.